botocore/data/cloudformation/2010-05-15/waiters-2.json

```diff
@@ -0,0 +1,197 @@
+{
+    "version": 2,
+    "waiters": {
+        "StackAvailable": {
+            "delay": 30,
```

Comment:
> What informs the 30 value you picked here?

Most Cloudformation APIs take many minutes to complete. For long-polling, 15 seconds are too frequent, 60 seconds are too long(FYW, Cloudfront and Redshift use 60 secs as polling intervals), and I thought 30 seconds are a good compromise.